### ISSUE TITLE
Prometheus I/O updates for WISE changes

### ISSUE THREAD

**Issue by RobBryce** [WISE-Developers/Prometheus-EOL-Issues#17]

**Comment by RobBryce** [comment]
WISE uses V2 of the FGMJ file format.
Prometheus uses V1 of the FGMJ file format.

V1 and V2 file formats aren't entirely compatible.  V2 is in a few cases, a super-set.  For example, although WISE will let you set burn conditions by sun rise and sun set, or by local noon, V1 only will interpret burn conditions off local midnight.

This step will define downgrade defaults when saving a V2 file as a V1 file.  These settings may belong in Prometheus, or in WISE, it's unclear (but likely WISE since it works with both file formats).

This action item is important for consistent/expected operation of Prometheus when using a project built using V2 of the file format.

**Comment by BadgerOnABike** [comment]
Defining these degradations will be useful for users.

As far as I understand for Prom to run V2 it needs to be run through WISE to generate a V1 file? Perhaps our notification of degradation can exist there.

**Comment by RobBryce** [comment]
Agreed, the rules that would need to be created in handling the conversion from V2 to V1 will itemize these changes.

Prometheus has no notion of V2 file formats.  You need to use WISE to convert V2 to V1.

**Comment by RobBryce** [comment]
Differences between Prometheus/V1 and WISE/V2 functionality are:
- specification of export files (suggest ignoring but retain to save properly)
- stop conditions that may mark the scenario as complete before the end date time (FI thresholds, RH, Precip, area burned, burn distance) (suggest ignoring but retain properly)
- sub-scenario specification (suggest ignoring, may be difficult to retain)
- burning conditions based on sunrise and sunset, or local noon rather than local midnight (this needs some discussion - it's easy to convert between local noon and local midnight, but conversion from sunrise and sunset won't be as easy)
- Prometheus has no notion of VAR's (assets) but could affect when a simulation stops (suggest retaining assets but break associations with scenarios so simulations aren't modified by something that the user cannot review or modify in the UI)
- Prometheus has no notion of targets (where to aim the windspeed to) that could modify the results of the simulation (suggest retaining targets but break associations with scenarios so simulation results aren't modified by something that cannot be reached by the UI)
- https://github.com/WISE-Developers/Project_issues/issues/197 may be useful for this work if approved, wouldn't be hard to integrate right now

For documentation:
https://github.com/WISE-Developers/Project_issues/issues/106 indicated that WISE should handle different timezone standards between WISE and Prometheus

Please review these differences and recommendations so I can introduce logic into Prometheus for each of these known differences in functionality and file format.

**Comment by RobBryce** [comment]
@nealmcloughlin stated to warn the user in the load warnings collection in this occurrence.  This will help but may not prevent the user from simply loading it and hitting the 'play' button, but that may be okay.

**Comment by lizzydchappy** [comment]
- specification of export files (suggest ignoring but retain to save properly)        **Please provide warning upon loading.**

- stop conditions that may mark the scenario as complete before the end date time (FI thresholds, RH, Precip, area burned, burn distance) (suggest ignoring but retain properly) **Agree**

- sub-scenario specification (suggest ignoring, may be difficult to retain) **Agree to ignore**

- burning conditions based on sunrise and sunset, or local noon rather than local midnight (this needs some discussion - it's easy to convert between local noon and local midnight, but conversion from sunrise and sunset won't be as easy) **Suggest this is not necessary to introduce to introduce into Prometheus. WISE has this functionality. Suggest a warning message in Prometheus to indicate different outputs.**

- Prometheus has no notion of VAR's (assets) but could affect when a simulation stops (suggest retaining assets but break associations with scenarios so simulations aren't modified by something that the user cannot review or modify in the UI) **Agree**

- Prometheus has no notion of targets (where to aim the windspeed to) that could modify the results of the simulation (suggest retaining targets but break associations with scenarios so simulation results aren't modified by something that cannot be reached by the UI) **Agree**

**Comment by RobBryce** [comment]
Gusting settings are also new so I'll suggest (as well) to retain but ignore, so no data is lost.
